gh-actions/common/build-debian/prepare-container.sh

```diff
@@ -14,6 +14,11 @@ EOF
 echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90aptyes
 apt update
 
+# Install ca-certificates to ensure that APT can access HTTPS repos
+apt install ca-certificates
+# Update package lists again, to ensure that we also have lists from HTTPS repos
+apt update
```

Comment:
Oh @adombeck , I would have preferred this to happen *only* if we have PPAs, that's not something we should do by default... The launchpad case is an exception, but otherwise we don't want any network access (it's in fact blocked by iptables) and these packages were installed only in specific cases (when used as input).

Reply:
> The launchpad case is an exception, but otherwise we don't want any network access (it's in fact blocked by iptables) and these packages were installed only in specific cases (when used as input).

I don't understand. We don't have to care about launchpad here because this script is only executed in out GitHub CI, not when building on launchpad, right? In our GitHub CI we do have network access, else all the code around this would also fail (e.g. the `apt update` right above or the `apt dist-upgrade` and `apt install eatmydata` below), right? Which packages were installed in specific cases?